.gitignore

```diff
@@ -290,4 +290,6 @@ __pycache__/
 package-lock.json
 test/src/tests/generated/*
 test/src/tests/vendor/*
-glide.lock
\ No newline at end of file
+glide.lock
```

Comment:
I see that you're not adding `glide.lock` to the ignore file, but I don't think it should have present in the first place. Mind removing it?

Reply:
Well, the glide up goal is to grab the latest version for the runtime. Dependencies for testing should not be tied to specific commits.

Reply:
Hmm, I guess I disagree. I think dependencies for testing should be handled the same way as others. Also, I think that `glide install` is the more typical command that should be run over `glide up`.